### DISCUSSION TITLE
W'bal

### DISCUSSION THREAD

**Discussion by nenadjev** [SauceLLC/sauce4zwift#43] [Q&A]
How to change default W'bal from 20Kj?

**Comment by mayfield** [comment]
Hi @nenadjev,

You can edit the W' from the athlete profile windows like this..

![image](https://user-images.githubusercontent.com/139316/196270804-3c5d6308-2d43-4c61-973a-ed608c0eec6f.png)

When you click the W' value it turns into an input.  Just change the value and hit Enter.

The athlete profile windows are accessed in various ways, but usually it is just by clicking into a group and then clicking on an athlete avatar/name.

**Reply by nenadjev** [reply]
Thank you, I've already found it👍

**Reply by PerBacks** [reply]
In the "riders nearby" window, I can see other riders W'. Is the value I see based on 20kJ or will I see each riders actually configured W', if they have modified it to another value than the 20kJ?

**Comment by pokeruno1** [comment]
I found out that I drain my KJ faster then I should.
For example i Ride 104w above CP for 121 seconds.
Should be 12584KJ but I drained 20100KJ.

Is the formula "(W-CP)* seconds=Drained KJ"
or is there another formula for short sprints?

**Reply by mayfield** [reply]
It uses the "Differential" algo which resulted from the second Skiba study..

https://github.com/SauceLLC/sauce4zwift/blob/b56fc803dc3a85445da5f4a383bab77dfd5ea3ee/shared/sauce/power.mjs#L758-L784

**Comment by pokeruno1** [comment]
ok, I see.
Is there another way to calculate your CP and KJ also?
Because it seems like a rider with high anaerobic capacity can get negative balance really easy even if both CP and KJ is set correct due to the common formula.
Or is it variance in how fast you increase your balance in break?

**Reply by mayfield** [reply]
I think a lot of folks use intervals.icu for doing predictive analysis.  It has several tools for analyzing your activity history and making predictions about CP and W' KJ levels.  There are a few schools of thought on it and a few different ways to make the predictions which I'm far from an expert on. :)
